subsys/fs/shell.c

```diff
@@ -10,6 +10,7 @@
 #include <zephyr/shell/shell.h>
```

Comment:
Commit message: 
nit: second and third sentences may be splitted to simpler :) 
"th3e" -> "the"